docker/sysdig/Dockerfile

```diff
@@ -31,14 +25,11 @@ RUN mkdir -p /tmp/dkms \
     && cd /tmp \
     && rm -fr /tmp/dkms
 
-COPY --from=builder /opt/sysdig /opt/sysdig
-RUN for b in /opt/sysdig/bin/*; do ln -s "$b" /usr/bin/$(basename "$b"); done \
-    && for b in /opt/sysdig/src/*; do ln -s "$b" /usr/src/$(basename "$b"); done
+COPY ./sysdig-release-${BUILD_VERSION}-*.rpm /
 
-COPY docker/sysdig/docker-entrypoint.sh /
+RUN rpm --nodeps --force -i ./sysdig-release-${BUILD_VERSION}-$(uname -p).rpm && rm /sysdig-release-*.rpm
 
-ENV HOST_ROOT /host
-ENV SYSDIG_HOST_ROOT /host
```

Comment:
Just double checking, have you verified that this is unused?

Reply:
These variables just moved upwards 😄